package.json

```diff
@@ -14,10 +14,11 @@
     "typecheck": "tsc",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
     "lint:css": "stylelint 'app/assets/stylesheets/**/*.scss' 'app/javascript/**/*.scss' 'app/components/*.scss'",
-    "lint:openapi": "redocly lint ./docs/attempts-api/openapi.yml",
+    "lint:openapi": "redocly lint public/attempts-api.json",
     "test": "mocha 'spec/javascript/**/**spec.+(j|t)s?(x)' 'app/javascript/packages/**/*.spec.+(j|t)s?(x)'",
     "normalize-yaml": "normalize-yaml",
     "generate-browsers-json": "./scripts/generate-browsers-json.js",
+    "generate-openapi-json": "redocly bundle -d docs/attempts-api/openapi.yml -o public/attempts-api.json --ext json --config config/redocly.yml",
```

Comment:
Do we want to make it publicly available via secure.login.gov if it isn't implemented yet?

Reply:
probably not! i was just sort of messing around to see what was possible, wasn't really intending on merging it with this available, but i should have noted that. (i do think we should keep the linting). 

(i also think, since there is a configuration view part of ssf that has specific endpoints, making this endpoint publicly available will be more for the dev docs to compile them as documentation.)